src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java

```diff
@@ -59,6 +60,7 @@
     private LocalDateTime lastUpdate;
 
     @ManyToOne
+    @JsonIgnore
```

Comment:
if we are including this as a response object then there will be a tight coupling between entity object and response object. I understand that there will be a lot of code needs to be written to create DTO but still a thought :)

Reply:
Let's discuss in call 👍

Reply:
Updated the code to use DTO instead of entity objects.